src/test/ui/const-generics/defaults/mismatch.rs

```diff
@@ -17,8 +17,6 @@ fn main() {
     //~^ Error: mismatched types
     let e: Example3::<7> = ();
     //~^ Error: mismatched types
-    // FIXME(const_generics_defaults): There should be a note for the error below, but it is
-    // missing.
     let e: Example4::<7> = ();
     //~^ Error: mismatched types
```

Comment:
you can test for the note without having to test for all notes by using

```suggestion
    //~^ Error: mismatched types
    //~| expected struct `Example4<7_usize>`
```

without `NOTE`

Reply:
Oh, you added it to all of them... in that case you can use NOTE before the message to ensure none get lost

Reply:
I spent a while trying to get `Note:` to work but couldn't :sweat_smile: Think ill just keep it how it is now lol the .stderr file will change if anything changes anyway so I think stuff getting dropped wouldnt be missed